openedx/core/djangoapps/discussions/views.py

```diff
@@ -69,6 +74,78 @@ def update(self, instance: LtiConfiguration, validated_data: dict) -> LtiConfigu
         return instance
 
 
+class LegacySettingsSerializer(serializers.BaseSerializer):
+    """
+    Serialize legacy discussions settings
+    """
+    class Meta:
+        fields = [
+            'allow_anonymous',
+            'allow_anonymous_to_peers',
+            'discussion_blackouts',
+            'discussion_link',
+            'discussion_sort_alpha',
+            'discussion_topics',
+        ]
+
+    def _fetch(self, descriptor) -> Iterator[tuple]:
+        """
+        Lookup all applicable fields from the course
+        """
+        for field in descriptor.fields.values():
+            if field.name not in self.Meta.fields:
+                continue
+            field_help = _(field.help)  # pylint: disable=translation-of-non-string
+            help_args = field.runtime_options.get('help_format_args')
+            if help_args is not None:
+                field_help = field_help.format(**help_args)
+            yield (field.name, {
+                'value': field.read_json(descriptor),
+                'help': field_help,
```

Comment:
Is this "help" stuff something we need?  In MFEs, field help text is generally just coded right into the frontend and localized there...

Reply:
The help text as-is might not be very useful right now. It is designed as help text for studio so expects that user's are entering raw JSON which might not be the case anymore. 

For instance, here is what it says for "discussion_sort_alpha":

> Enter true or false. If true, discussion categories and subcategories are sorted alphabetically. If false, they are sorted chronologically by creation date and time.

The above might not be useful in the MFE where there will be a toggle instead. I'm not sure if it will be possible to have a generic enough help text that applies across both.